config/mcp_config.py

```
@@ -0,0 +1,75 @@
+                    "command": "python3",
+                    "args": [str(Path(__file__).parent.parent / "mcp_server" / "main.py")],
+                    "env": {}
+                }
+            },
+            "tools": {
+                "enabled": ["code_analyzer", "protocol_validator", "self_corrector"],
+                "timeout": 30
+            }
+        }
+        
+    def get_config(self) -> Dict[str, Any]:
```

Ruff (0.11.9)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)